test/framework/src/main/java/org/elasticsearch/index/mapper/MapperTestCase.java

```diff
@@ -205,35 +222,39 @@ protected boolean supportsMeta() {
         return true;
     }
 
+    protected void metaMapping(XContentBuilder b) throws IOException {
```

Comment:
This is a pain, and the really annoying thing is that it's hard to fix these inconsistencies because of BWC.  I need to think some more about it.